release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2049,7 +2049,7 @@ With this update, downloaded CRLs are now tracked by the CA that distributes the
 +
 With this update, when creating a `DNSRecord` custom resource (CR) for a gateway listener, the Ingress Operator now sets the `DNSRecord's` DNS management policy to `Unmanaged` if its domain is outside of the cluster's base domain. As a result, the Ingress Operator no longer attempts to publish records, and no longer logs the `failed to publish DNS record to zone` error. (link:https://issues.redhat.com/browse/OCPBUGS-10875[*OCPBUGS-10875*])
 
-* Previously, `oc explain route.spec.tls.insecureEdgeTerminationPolicy` documented the incorrect possible options that could be confusing some some users. With this release, the API documentation has been updated so that it shows the correct possible options for `insecureEdgeTerminationPolicy`. This is an API documentation fix only. (link:https://issues.redhat.com/browse/OCPBUGS-11393[*OCPBUGS-11393*])
+* Previously, `oc explain route.spec.tls.insecureEdgeTerminationPolicy` documented the incorrect possible options that could be confusing to some users. With this release, the API documentation has been updated so that it shows the correct possible options for `insecureEdgeTerminationPolicy`. This is an API documentation fix only. (link:https://issues.redhat.com/browse/OCPBUGS-11393[*OCPBUGS-11393*])
```

Comment:
```suggestion
* Previously, `oc explain route.spec.tls.insecureEdgeTerminationPolicy` command documented the incorrect possible options that could be confusing to some users. With this release, the API documentation has been updated so that it shows the correct possible options for `insecureEdgeTerminationPolicy` field. This is an API documentation fix only. (link:https://issues.redhat.com/browse/OCPBUGS-11393[*OCPBUGS-11393*])
```

Reply:
I realize this is just targeting the typos but that might help clarify things a little. Instead of documented you could say showed as well.